### PR TITLE
fix broken links in markdown files

### PR DESCRIPTION
**Pull request by Myasuka**
Currently, some markdown files conatin links which are broken due to wrong file locations.
Please refer to my changed files. 

Except those broken links, I add `./` prefix to links in markdown files to stay code style the same.

BTW, I also remove extra spaces in these markdown files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,9 +116,9 @@ Refer to the [docs folder](docs) for design overview and other info on Mobius
 |Run Mobius Shell |<ul><li>[Local](notes/mobius-shell.md#run-shell)</li><li>[YARN](notes/mobius-shell.md#run-shell)</li></ul> | Not supported yet |
 
 ### Useful Links
-* [Configuration parameters in Mobius](/notes/configuration-mobius.md)
-* [Troubleshoot errors in Mobius](/notes/troubleshooting-mobius.md)
-* [Debug Mobius apps](/notes/running-mobius-app.md#debug-mode)
+* [Configuration parameters in Mobius](./notes/configuration-mobius.md)
```
Comment (tawan0109) [inline README.md]: Just curious how did you find this link broken? Quick check shows that this link (and some other links with similar changes) is not broken on GitHub.

**Comment by tawan0109** [comment]
LGTM except one inline comment.

File: README.md (hunk 1 above)
Comment (Myasuka) [inline README.md]: The real broken links are those in `notes/CONTRIBUTING.md`, `notes/mobius-shell.md` and the line 62 of `notes/windows-instructions.md`, I add other links of markdown files with prefix `./` to make the link code style stay the same.

File: notes/linux-instructions.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ If you are using CentOS, Fedora, or similar Linux distributions or OS X, follow
   * Update CSharpWorkerPath setting in Mobius application config (refer to the config files used in Mobius examples like the [config for with Pi example](https://github.com/skaarthik/Mobius/blob/linux/examples/Batch/pi/App.config#L61)) to point to [CSharpWorker.sh.exe](./linux-csharpworker-prefix-script.md) (make sure to set the correct value appropriate for the Spark mode to be used)
 
 ### Mobius in Azure HDInsight Spark Cluster
-* Mono version available in HDInsight cluster is 3.x. Mobius [requires](/notes/linux-instructions.md#prerequisites) 4.2 or above. So, Mono has to be upgraded in HDInsight cluster to use Mobius.
+* Mono version available in HDInsight cluster is 3.x. Mobius [requires](./linux-instructions.md#prerequisites) 4.2 or above. So, Mono has to be upgraded in HDInsight cluster to use Mobius.
```
Comment (Myasuka) [inline notes/linux-instructions.md]: BTW, the link of line 55 is also broken.

File: README.md (hunk 1 above)
Comment (tawan0109) [inline README.md]: I see.  Please also state this change (code style) in the PR description.

**Pull request merged**